### ISSUE TITLE
Signing in with AAD account requiring MFA does not work

### ISSUE THREAD

**Issue by judyanndixon** [MicrosoftDocs/vscodespaces#236]
I am using Chrome Version 78.0.3904.108 (Official Build) (64-bit) and Windows 10 1903.  My AAD account requires MFA.  If I am signed out of this account and try to sign into VS Online I am not prompted for MFA.  It does load my already created environments but when I try to create a new billing plan, it does not list any subscriptions instead it allows me to create a new Azure account.

If I first log into the Azure portal using the same account and do the MFA which is required, I can then log into VS Online and when I attempt to create a new billing plan, the list of subscriptions is populated from the default tenant of the account and there is no option to create a free Azure Account.

**Comment by nikmd23** [comment]
@legomushroom, have you been able to repro this issue? I have not been able to yet.

**Comment by legomushroom** [comment]
That is the issue that if a user is part of multiple tenants, we fall back to using the first one without the ability for them to change that. We aware of the issue but it is currently blocked by some auth changes in the queue.

**Comment by lostintangent** [comment]
Closing this as a duplicate of #550.
